code/API_definitions/device-status.yaml

```diff
@@ -30,9 +31,39 @@ servers:
         description: Base path for the device status API
 tags:
   - name: Device roaming status
-    description: Operations to validate device roaming status and country information
+    description: Operations to get device roaming status and country information on the fly
+  - name: Device roaming status subscription
+    description: Operation to manage subscription on device status roaming event
+  - name: Device status subscription
+    description: Operation to manage subscription on device status event (roaming, connectivity)
+  - name: Notification Callback
+    description: Notification received on subscription listener side
 paths:
-   /roaming:
+  /notifications:
+    post:
+      tags:
+        - Notification Callback
+      summary: "Subscription notification callback"
+      operationId: postNotification
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/Notification"
+            examples:
+              ROAMING_STATUS:
+                $ref: '#/components/examples/ROAMING_STATUS'  
+              ROAMING_CHANGE_COUNTRY:
+                $ref: '#/components/examples/ROAMING_CHANGE_COUNTRY' 
+              ROAMING_ON:
+                $ref: '#/components/examples/ROAMING_ON'
+              ROAMING_OFF:
+                $ref: '#/components/examples/ROAMING_OFF'
+      responses:
```

Comment:
From the guideline:

_Error definition are described in this guideline applies for event notification.
Following Error code must be present:_

    for POST: 400, 401, 403, 500, 503

Reply:
Fixed